WordPress/src/main/java/org/wordpress/android/util/QuickStartUtils.kt

```diff
@@ -0,0 +1,50 @@
+package org.wordpress.android.util
+
+import android.content.Context
+import android.support.v4.content.ContextCompat
+import android.support.v4.graphics.drawable.DrawableCompat
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.ForegroundColorSpan
+import android.text.style.ImageSpan
+import org.wordpress.android.R
+
+class QuickStartUtils {
+    companion object {
+        /**
+         * Formats the string, to highlight text between %1$s and %2$s with specified color, and add an icon
+         * in front of it if necessary
+         *
+         * @param context Context used to access resources
+         * @param messageId resources id of the message to display
+         * @param iconId resource if of the icon that goes before the highlighted area
+         */
+        @JvmStatic
+        fun stylizeQuickStartPrompt(context: Context, messageId: Int, iconId: Int): Spannable {
+            val message = context.resources.getString(messageId)
+            val icon = ContextCompat.getDrawable(context, iconId)
+
+            val highlightColor = ContextCompat.getColor(context, R.color.blue_light)
+
+            val startOfHighlight = message.indexOf("%1${'$'}s")
+            var clearString = message.replace("%1${'$'}s", if (icon != null) " " else "")
+            val endOfHighlight = clearString.indexOf("%2${'$'}s")
+            clearString = clearString.replace("%2${'$'}s", "")
```

Comment:
We still need to know the boundaries of a text that we need to highlight, so I left them as a "markers", and `String.replace()` takes care of removing them later.

I just recently realized that the original plan was to use `Html.fromHtml()`.  Maybe it makes sense to get back to it and place the icon a the beginning of the `ForegroundColorSpan` span produced by `Html.fromHtml()`. The only downside to this is that we hardcode the color in XML. What do you think?

Reply:
> We still need to know the boundaries of a text that we need to highlight, so I left them as a "markers"

Ahh, okay.

> What do you think?

Yeah, there doesn't seem like an easy way to add the icon and span to snackbar messages.  Replacing the `%1$s` and `%2$s` feels hacky.  Hardcoding the color in XML isn't ideal.  Whichever way you like works for me.

Reply:
So I went for the third option. Using span tags and `Html.fromHtml()` but only to determine the boundaries of the highlighted area. We ignore the color in XML and set it programmatically.

Ready for another round!

Reply:
> We ignore the color in XML and set it programmatically.

Should we change the color value in the starting span tag string resource to something other than `#78dcfa` since it's being ignored?  Someone may try to change that value in the future expecting it to update the highlight color and get confused trying to figure out where that value is actually set.  Do you think changing it to a dummy value like `#000000` or `#ffffff` would help?